### PR TITLE
Acsone addons newsletter with distribution list

### PR DESCRIPTION
**Pull request by JonathanNEMRY**
[mass_mailing_distribution_list] Introduce newsletter concept for mass mailing using distribution list
[distribution_list] Review interface of distribution list due to new m2m  lists of newsletter feature

### CODE REVIEW

File: mass_mailing_distribution_list/mass_mailing.py [hunk 1]
```diff
@@ -27,6 +27,12 @@
 #
 ##############################################################################
 from openerp.osv import orm, fields
+from openerp.tools.translate import _
+
+MSG_OK = _("<p>Unsubscribe done successfully.</p>")
```
Comment (Olivier-LAURENT) [inline mass_mailing_distribution_list/mass_mailing.py]: Which language will be choose when replying these MSGs ? Uid's language is not the right language in this context.

File: mass_mailing_distribution_list/mail_mail.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            }
+            url = urlparse.urljoin(
+                base_url, 'mail/newsletter/%(mailing_id)s/'
+                          'unsubscribe?%(params)s' % {
+                              'mailing_id': mail.mailing_id.id,
+                              'params': urllib.urlencode(vals)
+                          }
+            )
+            return '<small><a href="%s">%s</a></small>' % (
+                url, msg or 'Click to unsubscribe')
```
Comment (Olivier-LAURENT) [inline mass_mailing_distribution_list/mail_mail.py:76]: Translation ? Ok if no solution for the previous comment.

**Comment by Olivier-LAURENT** [comment]
:+1:

**Comment by laetitia-gangloff** [comment]
:+1:

**Pull request merged**